.husky/pre-commit

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env sh
+. "$(dirname -- "$0")/_/husky.sh"
+
+npx lint-staged
```

Comment:
nice so when this detects issues, we can manually run lint:fix right? then attempt to commit again?

Reply:
Right.